server/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const PORT = 2000 || process.env.PORT;
+const db = require('../db/index.js');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
+app.use(express.static('client/dist'))
+app.use(bodyParser.urlencoded( { extended: true } ))
+app.use(bodyParser.json())
+app.use(cors());
+
+app.listen(PORT, () => {
+  console.log(`Listening on port ${PORT}`);
+})
+
+// get request example
+app.get('/movies', (req, res) => {
```

Comment:
have you verified using Postman/curl that this route returns the db data as expected?

Reply:
Brock, 
Yes I have tested with Postman and indeed I receive the db data as expected. Great question.